server/services/notificationService.js

```diff
@@ -1,7 +1,7 @@
 const { NotifyClient } = require('notifications-node-client')
 const moment = require('moment')
 const { email } = require('../config')
-const { stringToHash } = require('../utils/utils')
+const { stringToHash } = require('../utils/hash')
```

Comment:
I moved this into separate module as part of the wrangling with int tests.. think its probably fine here